modules/ROOT/pages/outbound-message-routing.adoc

```diff
@@ -14,7 +14,14 @@ When messages are received by the endpoint, Partner Manager routes the payload t
 ** Relative path for HTTP(s) endpoints
```

Comment:
These look like they should be numbered actions rather than bullets.
Also, GLOBALLY replace "HTTP(s)" or "HTTP(S)" by naming both: HTTP or HTTPS.

Reply:
Fix capitalization typo of "identifies" in line 13.

Reply:
Done - will do global check in a different PR